### ISSUE TITLE
How I can make confusion matrix

### ISSUE THREAD

**Issue by nagaokatakuya** [experiencor/basic-yolo-keras#122]
I finished training my dataset(20 classes) about 4000 images.
Then, to make sure right classification by this model, I tried to make confusion matrix by reference to "predict.py", "frontend.py" and "preprocessing.py". 
But in basic-yolo-keras, formats of input datas are different from these in normal keras.
So, it is difficult to make confusion matrix.
I'd like some advice.

**Comment by experiencor** [comment]
Hi @nagaokatakuya! Very sorry for missing out this issue. If you want to evaluate the performance of the object detection model, you can use mAP score instead of confusion matrix (https://github.com/experiencor/basic-yolo-keras/issues/27)

**Comment by nagaokatakuya** [comment]
Thanks, but I implemented matrices similar to confusion matrix using annotation file and "predict.py". Refer to center x and y coordinations of boxes of annotation files, I can search for the boxes predicted near to the boxes written in annotation files and count labels predicted about the boxes. The metrices are reasonably good to evaluate the detection model.

mAP score would be helpful to evaluate, the next I will implement it.

**Comment by karan2808** [comment]
Hey do you mind sharing your code?

**Comment by zmlim** [comment]
Hi @karan2808  @nagaokatakuya  May I know if you guys had any progress with the confusion matrix code?
